src/main/java/org/folio/inventory/dataimport/handlers/matching/loaders/AbstractLoader.java

```diff
@@ -49,9 +59,16 @@ public AbstractLoader(Vertx vertx) {
             if (collection.totalRecords == 1) {
               loadResult.setValue(mapEntityToJsonString(collection.records.get(0)));
             } else if (collection.totalRecords > 1) {
-              String errorMessage = String.format("Found multiple records matching specified conditions. CQL query: [%s].\nFound records: %s", cql, Json.encodePrettily(collection.records));
-              LOG.error(errorMessage);
-              future.completeExceptionally(new MatchingException(errorMessage));
+              if (canProcessMultiMatchResult(eventPayload)) {
+                LOG.info("Found multiple records by CQL query: [{}]. Found records IDs: {}", cql, mapEntityListToIdsJson(collection.records));
+                loadResult.setEntityType(MULTI_MATCH_IDS);
+                loadResult.setValue(mapEntityListToIdsJson(collection.records));
+              } else {
+                String errorMessage = String.format("Found multiple records matching specified conditions. CQL query: [%s].%nFound records: %s", cql, Json.encodePrettily(collection.records));
```

Comment:
General question, do we care if customer information gets posted to logs?

Reply:
the records don't contain sensitive data, so suppose they can be posted